packages/dds/tree/src/simple-tree/leafNodeSchema.ts

```diff
@@ -35,14 +40,21 @@ import type { JsonCompatibleReadOnlyObject } from "../util/index.js";
 export class LeafNodeSchema<Name extends string, const T extends ValueSchema>
 	implements
 		TreeNodeSchemaNonClass<Name, NodeKind.Leaf, TreeValue<T>, TreeValue<T>>,
-		SimpleLeafNodeSchema
+		SimpleLeafNodeSchema,
+		TreeNodeSchemaCorePrivate
 {
 	public readonly identifier: Name;
 	public readonly kind = NodeKind.Leaf;
 	public readonly info: T;
 	public readonly implicitlyConstructable = true as const;
 	public readonly childTypes: ReadonlySet<TreeNodeSchema> = new Set();
 	public readonly childAnnotatedAllowedTypes: readonly NormalizedAnnotatedAllowedTypes[] = [];
```

Comment:
This property is being removed in the other node types so I assume it should also be removed here?

```suggestion
```